src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -0,0 +1,26 @@
+package cromwell.engine.db
+
+import java.util.Date
+
+import cromwell.binding.{Call, WdlSource}
+import cromwell.engine.store.SymbolStoreEntry
+import cromwell.engine.{WorkflowId, WorkflowState}
+
+trait DataAccess {
+  // Will stamp the start_dt column with DateTime.Now.
+  // Will stamp the WorkflowState as Submitted.
+  def createWorkflow(id: WorkflowId, wdlUri: String, wdlSource: WdlSource, jsonInputs: String, symbols: Seq[SymbolStoreEntry]): Unit
```

Comment:
I was wondering the same thing!  Actually I think this is obviated by the workflow_aux table (or whatever it's called now).  I had this is my original SQL layout as a way to get at the WDL source, but since we're storing it in its entirety, this isn't necessary.